src/TypeScript/DefinitionWriter.cpp

```diff
@@ -91,7 +91,7 @@ void DefinitionWriter::visit(InterfaceMeta* meta)
 
     _buffer << std::endl << "\tdeclare class " << meta->jsName << getTypeArgumentsStringOrEmpty(meta);
     if (meta->base != nullptr) {
-        _buffer << " extends " << localizeReference(*meta->base);
+        _buffer << " extends " << localizeReference(*meta->base) << getTypeArgumentsStringOrEmpty(meta->base);
```

Comment:
Although type parameters can be renamed, this heuristic is working for the iOS SDK.